README.md

```diff
@@ -178,13 +178,13 @@ secrets := []string{"mySecretKey"}
 // OR regexp keys
 // secrets := []*regexp.Regexp{regexp.MustCompile("mySecretKey")}
 
-blacklist := make([]interface{}, len(secrets))
+blocklist := make([]interface{}, len(secrets))
 for i, v := range secrets {
-	blacklist[i] = v
+	blocklist[i] = v
 }
 
 opts := gobrake.NotifierOptions{
-	KeysBlacklist: blacklist,
```

Comment:
When this change is applied, wouldn't `KeysBlacklist` be a no-op? Meaning if a notifier is upgraded, the old blacklist configuration would be broken.

Reply:
This addition is meant to handle that. ie: look for the deprecated `KeysBlacklist` if the new `KeysBlocklist` is not set. 

https://github.com/airbrake/gobrake/pull/141/files#diff-973a1a56d0cc9506ee822e801b41e635R96-R98

That is what this test should be verifying (let me know if you spot any issues though):

https://github.com/airbrake/gobrake/pull/141/files#diff-5a7ab9d3893017f739c4ecf882378f7dR329-R392

Or are you saying that this readme section should mention the deprecated option?

Reply:
It makes sense but when I read the code it's not obvious. I had to read the linked code multiple times before I could understand how it works 😅

Why not to check if `len(opt.KeysBlacklist) > 0` and then forcefully assign to  `opt.KeysBlocklist`. Check this example in airbrake-ruby https://github.com/airbrake/airbrake-ruby/blob/0658c7c26dbed654f5582594f7aa21fbe251263f/lib/airbrake-ruby/config.rb#L188-L195

This way the code is more obvious. Does it make sense?

Reply:
Added an update that:
- Swaps the conditional to make it clearer. Like you said, it makes more sense to check if they're using the deprecated option, then assign it.
- Adds a log message mentioning that the option is deprecated.

Reply:
The log message is very useful but it is being printed on `go test`, which isn't good. We should temporarily swap logger during the test that invokes this message, so that we don't pollute stdout. Here's how we can do it:

https://github.com/airbrake/gobrake/blob/173654ce122fca2e86611ce3597735e4fbcb745f/notifier_test.go#L279-L301

Just set a dummy logger that prints to `/dev/null` while the test is running and then set the original logger back.

Reply:
This is how you can create a dummy logger: `log.New(io.Discard, "gobrake: ", log.LstdFlags|log.Lshortfile)`